nxrm-ha/README.md

```diff
@@ -169,6 +169,97 @@ AWS Secret Manager is disabled by default. If you would like to store your datab
 - You'll need an IAM role with necessary permissions and associate that IAM role with the service account used by your pods:
 - See [External secrets operator EKS service account credentials](https://external-secrets.io/latest/provider/aws-secrets-manager/#eks-service-account-credentials) for more details.
 
+##### Cert-Manager Configuration
```

Comment:
idk if this is only in gh preview but the readme is kind of broken , the table seems to cut at some point

Reply:
After checking , it seems like there is a line break in the table section , probably needs to get fixed